test/functional/lua/vim_spec.lua

```diff
@@ -3360,4 +3360,31 @@ describe('vim.keymap', function()
     eq(1, exec_lua[[return GlobalCount]])
   end)
 
+  it('exists() can check a lua function', function()
+    command('lua _G.test = function() print("hello") end')
+    eq(true, exec_lua[[
+      return vim.fn.exists('v:lua.test') == 1
+    ]])
+
+    eq(true, exec_lua[[
+      return vim.fn.exists('v:lua.require("vim.lsp").with') == 1
```

Comment:
`with` is deprecated...
```suggestion
      return vim.fn.exists('v:lua.require("vim.mpack").decode') == 1
```

Reply:
mpack is stdlib function..I don't think it can used in there.

Reply:
use something other than `with`

Reply:
However using `v:lua.require'mpack'.decode` should work, as `require'mpack'.decode` exists.

Reply:
tested it with current implementation. `require'mpack'.decode` is work . `vim.mpack` shouldn't work i think